scripts/core/EXP-COMMAND-BUFFER.rst

```diff
@@ -173,7 +173,14 @@ ${X}_DEVICE_INFO_COMMAND_BUFFER_UPDATE_SUPPORT_EXP query.
 Updating kernel commands is done by passing the new kernel configuration
 to ${x}CommandBufferUpdateKernelLaunchExp along with the command handle of
 the kernel command to update. Configurations that can be changed are the
-parameters to the kernel and the execution ND-Range.
+kernel handle, the parameters to the kernel and the execution ND-Range.
+
+To update the kernel handle, it is necessary to first register the new
+kernel handle before the command-buffer is finalized. This can be done
```

Comment:
Nit: I think this could be clearer that they must be registered/provided in the call to add the kernel command to the command-buffer, it reads a bit ambiguously here to me saying it's just before finalization.

Reply:
I updated the wording a bit and removed the finalize mention. Hopefully it reads less ambiguously :crossed_fingers: